src/components/menu-surface/menu-surface.scss

```diff
@@ -0,0 +1,16 @@
+@import '../../style/variables';
+@import "@limetech/mdc-menu-surface/mdc-menu-surface";
+
+.mdc-menu-surface {
+    width: 100%;
+    max-height: 100%;
+}
+
+.mdc-menu-surface--scrim {
+    position: fixed;
+    width: 100%;
+    height: 100%;
+    top: 0;
+    left: 0;
+    z-index: 5;
```

Comment:
Just an idea; I don't know how much we use _z-index_ but it might be useful to put the different levels in vars to avoid unexpected behaviour or "z-index inflation" :)

Reply:
@joelricci We usually do use variables for our z-indexes, at least in the webclient. I'm not sure in how many instances we set z-index in lime-elements though.

I guess we might even want to use a css-variable, so the default can be overridden by the consumer…?

Reply:
I kind of forgot about the z-index :smile: It might not be needed. I added it as I was testing in the docz app and at first the dropdown would appear underneath the navigation menu. Material has some z-indexes defined for different levels so we should probably comply to those